spec/pg_easy_replicate_spec.rb

```diff
@@ -21,7 +21,7 @@
             { name: "max_replication_slots", setting: "10" },
             { name: "max_wal_senders", setting: "10" },
             { name: "max_worker_processes", setting: "8" },
-            { name: "wal_level", setting: "logical" },
+            { name: "wal_level", setting: "replica" },
           ],
```

Comment:
Test was failing as this was returning as replica, so updated to make tests pass. I assume it was a config typo in the test setup but LMK if this indicates something broken.